edit_gpx/GPX.js

```diff
@@ -0,0 +1,243 @@
+        html+= '        </div>';
+        html+= '        <div class="controls">';
+        html+= '            <label class="inline-block" for="lineToggle">';
+        html+= '                <input type="radio" name="type" value="line" id="lineToggle" />';
+        html+= '            draw line</label>';
+        html+= '        </div>';
+        html+= '        <div class="controls">';
+        html+= '            <label class="inline-block" for="polygonToggle">';
+        html+= '                <input type="radio" name="type" value="polygon" id="polygonToggle" />';
+        html+= '            draw polygon</label>';
+        html+= '        </div>';
+        html+= '        <div class="controls">';
+        html+= '            <label class="inline-block" for="modifyToggle">';
+        html+= '              <input type="radio" name="type" value="modify" id="modifyToggle"/>';
+        html+= '              Modifier les données';
+        html+= '            </label>';
+        html+= '            <div class="controls">';
+        html+= '                <div hidden>';
+        html+= '                    <input id="createVertices" type="checkbox" checked';
+        html+= '                           name="createVertices" />';
+        html+= '                    <label for="createVertices">allow vertices creation</label>';
+        html+= '                </div>';
+        html+= '            </div>';
+        html+= '        </div>';
+        html+= '    </div>';
+        html+= '    <button type="button" id="export" style="margin-top:10px;" class="btn btn-light">Exporter</button><br>';
+        html+= '</div>';
+
+        // Distance ou temps
+        //html+= '<div class="control-group">';
+
+        //html+= '</div>';
+
+
+        // Reverse
+        //html+= '<div class="control-group">';
+
+
+        html+= '</div>';
+
+        html+= '</form>'
+        html+= '</div>'
+
+        lizMap.addDock(
+            'GPX',
+            'Edit your GPX file',
+            'minidock',
+            html,
+            'icon-road'
+        );
+
+        initGpxView();
+
+    }
+});
+
+function initGpxView() {
+    var map, vectors, controls, fileName;
+    var gpxLayer;
+    var defaultFillColor = "yellow";
+    var defaultStrokeColor = "orange";
+    var defaultStrokeWidth = 5;
+    var selectFillColor = "orange";
+    var selectStrokeColor = "red";
+    var selectStrokeWidth = 5;
+    var pointRadius = 6;
+
+    map = lizMap.map;
+    var myStyles = new OpenLayers.StyleMap({
+    "default": new OpenLayers.Style({
+        pointRadius: pointRadius,
+        fillColor: defaultFillColor,
+        strokeColor: defaultStrokeColor,
+        strokeWidth: defaultStrokeWidth
+    }),
+    "select": new OpenLayers.Style({
+        pointRadius: pointRadius,
+        fillColor: selectFillColor,
+        strokeColor: selectStrokeColor,
+        strokeWidth: selectStrokeWidth
+    })
+    });
+
+    vectors = new OpenLayers.Layer.Vector('default name', {
```

Comment:
Can you named the OpenLayers vector layer as `gpx` ?
```suggestion
    vectors = new OpenLayers.Layer.Vector('gpx', {
```